hack/ci/await-pr-merge.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset  # treat unset variables as an error and exit immediately.
+set -o errexit  # exit immediately when a command fails.
+set -E          # needs to be set if we want the ERR trap
+set -o pipefail # prevents errors in a pipeline from being masked
+
+# Expected environment variables:
+# PR_URL - Number of the PR with the changes to be merged
+
+# wait until the PR is merged.
+while true ; do
+  pr_state=$(gh pr view ${PR_URL} --json state --jq '.state')
+  if [ "$pr_state" == "CLOSED" ]; then
+    echo "ERROR! PR has been closed!"
+    exit 1
+  elif [ "$pr_state" == "MERGED" ]; then
+    echo "PR has been merged!"
+    exit 0
+  fi
+  echo "Waiting for ${PR_URL} to be merged"
```

Comment:
When will this time out?

Reply:
I have added it [here](https://github.com/kyma-project/eventing-manager/pull/543/commits/4ef9aad7be79aab20d1f3c6324ad2c99a91b2577).